webapp/test/LoginForm.spec.js

```diff
@@ -0,0 +1,275 @@
+import '@testing-library/jest-dom'
+import 'regenerator-runtime'
+
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+
+import VueApollo from 'vue-apollo'
+import { createMockClient } from 'mock-apollo-client'
+import login from '@/apollo/mutations/login'
+
+import Vuex from 'vuex'
+
+import LoginForm from '@/components/LoginForm'
+import { actions, getters, mutations, state } from '@/store/auth'
+import { InMemoryCache } from '@apollo/client'
+
+import jwt_decode from 'jwt-decode'
+
+jest.mock('jwt-decode')
+
+describe('LoginForm', () => {
+  const localVue = createLocalVue()
+  localVue.use(VueApollo)
+  localVue.use(Vuex)
+
+  let wrapper, mockClient, apolloProvider, store
+
+  const createLoginForm = async (handlers, appOptions) => {
+    mockClient = createMockClient({ cache: new InMemoryCache() })
+    apolloProvider = new VueApollo({
+      defaultClient: mockClient
+    })
+
+    const requestHandlers = {
+      loginHandler: null,
+      ...handlers
+    }
+    mockClient.setRequestHandler(
+      login,
+      requestHandlers.loginHandler
+    )
+
+    store = new Vuex.Store({
+      modules: {
+        auth: {
+          namespaced: true,
+          state,
+          getters,
+          actions,
+          mutations
+        }
+      }
+    })
+
+    wrapper = shallowMount(LoginForm, {
+      localVue,
+      apolloProvider,
+      store,
+      stubs: {
+        NuxtLink: true,
+      },
+      ...appOptions
+    })
+    await wrapper.vm.$nextTick()
+  }
+
+  afterEach(() => {
+    wrapper.destroy()
+    mockClient = null
+    apolloProvider = null
+    store = null
+  })
+
+  describe('when no user is logged in', () => {
+    it('should show the login form', async () => {
+      await createLoginForm()
+      await wrapper.vm.$nextTick()
```

Comment:
```suggestion
```
This cannot be necessary anymore, if you have `await` in the method above :point_up:

Reply:
Again, it shouldn't be necessary, but somehow is :man_shrugging: